grafana/provisioning/dashboards/CHT/cht_partnerships_replication.json

```
@@ -0,0 +1,638 @@
+                "result": {
+                  "text": "N/A"
+                }
+              },
+              "type": "special"
+            }
+          ],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "#C4162A",
+                "value": null
+              },
+              {
+                "color": "rgba(237, 129, 40, 0.89)",
+                "value": 70
+              },
+              {
+                "color": "#299c46",
+                "value": 90
+              }
+            ]
+          },
+          "unit": "none"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 2,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": 51,
+      "links": [],
+      "maxDataPoints": 100,
+      "options": {
+        "colorMode": "background",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "horizontal",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "10.0.2",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "fa8f4e3c-53cf-499d-89dd-8d6eaca0a756"
+          },
+          "editorMode": "code",
+          "expr": "(\n        (   \n                sum(cht_api_http_request_duration_seconds_bucket{instance=~\"$instance\",le=\"90\",route=~\".*/get-ids\",code=~\"^2..$\"})\n                + (\n                        sum(cht_api_http_request_duration_seconds_bucket{instance=~\"$instance\",le=\"180\",route=~\".*/get-ids\",code=~\"^2..$\"})\n                        - sum(cht_api_http_request_duration_seconds_bucket{instance=~\"$instance\",le=\"90\",route=~\".*/get-ids\",code=~\"^2..$\"})\n                ) / 2\n        ) / sum(cht_api_http_request_duration_seconds_count{instance=~\"$instance\",route=~\".*/get-ids\",code=~\"^2..$\"})\n) * 100",
+          "legendFormat": "score",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Replication Apdex",
+      "type": "stat"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "fa8f4e3c-53cf-499d-89dd-8d6eaca0a756"
+      },
+      "fieldConfig": {
+        "defaults": {
+          "links": []
+        },
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 5,
+        "w": 12,
+        "x": 12,
+        "y": 0
+      },
+      "hiddenSeries": false,
+      "id": 32,
+      "legend": {
+        "alignAsTable": true,
+        "avg": false,
+        "current": false,
+        "max": true,
+        "min": false,
+        "rightSide": true,
+        "show": true,
```

```suggestion
        "show": false,
```

Minor, but since there is only one data series on this panel (for `/get-ids`), we do not really need the legend on the side....

---

I've intended to see two panels - one for /initial-replications/get-ids and one for /replication/get-ids. I'll give this another test, but can you share a screenshot showing what you're hoping to change?

---

Ah, I think when I was looking at this, I though there would only be one time-series displayed. Now with your latest fake code, I can see two and so it makes sense to have the legend.